es-ds-docs/components/ds-code-block.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <!-- eslint-disable vue/no-v-html -->
+    <div class="prism">
```

Comment:
A minor thing: The prism plugin doesn't seem to be kicking in for the code samples. I don't see line numbers or the Copy/View Source buttons when I hover over the code samples.

Reply:
Ah good catch - I saw syntax highlighting and code and figured that was it.

I've created a ticket to fix Prism (https://energysage.atlassian.net/browse/CED-1764).